football_dictionaries/assignment_2.py

```diff
@@ -1,2 +1,13 @@
+from .assignment_1 import players_as_dictionaries
+
 def players_by_position(squads_list):
-    pass
+    position_dict = players_as_dictionaries(squads_list)
```

Comment:
I would name this something different, since players_as_dictionaries() returns an array of player objects. My first assumption when looking at the variable position_dict leads me to think that this variable holds just a dictionary of positions. I guess it would be better if I just showed you what i'm visualizing when I think about the variable.

position_dict = { ...'some_label': 'some_info'  }

when it's actually position_dict = [{}, {}, {}, ...]